### ISSUE TITLE
Release Note of v9.0.0 includes changes of v10.0

### ISSUE THREAD

**Issue by eseokoh** [vitessio/vitess#8212]
#### Overview of the Issue

https://github.com/vitessio/vitess/releases/tag/v9.0.0 includes some changes that are not shipped as v9.0.0.

I guess they are merged into `master` branch after branching `release-9.0` and accidentally tagged as `v9.0`. 
Please confirm the following list:

---

#### VTGate / MySQL compatibility

- VTGate: Planner refactoring [#7103](https://github.com/vitessio/vitess/pull/7103)
- VTGate: Migrate vtctlclient InitShardMaster => vtctldclient InitShardPrimary [#7220](https://github.com/vitessio/vitess/pull/7220)
- VTGate: Add Planning and Parsing Support for Truncate, Rename, Drop Index and Flush [#7242](https://github.com/vitessio/vitess/pull/7242)
- VTGate: Added default databases when calling 'show databases' [#7256](https://github.com/vitessio/vitess/pull/7256)
- VTGate :Rremoved resolver usage from StreamExecute [#7281](https://github.com/vitessio/vitess/pull/7281)
- VTGate: Adding a MySQL connection at Vtgate to run queries on it directly in case of testing mode [#7291](https://github.com/vitessio/vitess/pull/7291)
- VTGate: Added vitess_version as variable [#7295](https://github.com/vitessio/vitess/pull/7295)
- VTGate: Gen4: Add Limit clause support [#7312](https://github.com/vitessio/vitess/pull/7312)
- VTGate: Gen4: Handling subquery in query graph [#7313](https://github.com/vitessio/vitess/pull/7313)
- VTGate: Addition of @@enable_system_settings [#7300](https://github.com/vitessio/vitess/pull/7300)

#### VReplication

- VReplication: Tablet throttle support "/throttle/check-self" available on all tablets [#7319](https://github.com/vitessio/vitess/pull/7319)

#### VTadmin

- VTadmin: Add vtctld proxy to vtadmin API, add GetKeyspaces endpoint [#7266](https://github.com/vitessio/vitess/pull/7266)
- VTadmin: [vtctld] Expose vtctld gRPC port in local Docker example + update VTAdmin README [#7306](https://github.com/vitessio/vitess/pull/7306)
- VTadmin: Add CSS variables + fonts to VTAdmin [#7309](https://github.com/vitessio/vitess/pull/7309)
- VTadmin: Add React Router + a skeleton /debug page to VTAdmin [#7310](https://github.com/vitessio/vitess/pull/7310)
- VTadmin: Add NavRail component [#7316](https://github.com/vitessio/vitess/pull/7316)
- VTadmin: Add Button + Icon components [#7350](https://github.com/vitessio/vitess/pull/7350)

- [vtctld] Migrate cell getters [#7302](https://github.com/vitessio/vitess/pull/7302)
- [vtctld] Migrate tablet getters [#7311](https://github.com/vitessio/vitess/pull/7311)
- [vtctld] Migrate GetSchema [#7346](https://github.com/vitessio/vitess/pull/7346)
- [vtctld] vtctldclient command pkg [#7321](https://github.com/vitessio/vitess/pull/7321)
- [vtctld] Add GetSrvVSchema command [#7334](https://github.com/vitessio/vitess/pull/7334)
- [vtctld] Migrate ListBackups as GetBackups in new vtctld server [#7352](https://github.com/vitessio/vitess/pull/7352)
- [vtctld] Migrate GetVSchema to VtctldServer [#7360](https://github.com/vitessio/vitess/pull/7360)

#### Examples/Tutorials

- Delete select_commerce_data.sql [#7245](https://github.com/vitessio/vitess/pull/7245)
- Docker/vttestserver: Add MYSQL_BIND_HOST env [#7293](https://github.com/vitessio/vitess/pull/7293)

#### Documentation

- Pull Request template: link to contribution guide [#7314](https://github.com/vitessio/vitess/pull/7314)

#### Build Environment Changes

- Github Actions CI Builds: convert matrix strategy for unit and cluster tests to individual tests [#7258](https://github.com/vitessio/vitess/pull/7258)
- Descriptive names for CI checks [#7289](https://github.com/vitessio/vitess/pull/7289)
- Add mysqlctl to docker images [#7326](https://github.com/vitessio/vitess/pull/7326)

---

- Add Update.AddWhere to mirror Select.AddWhere [#7277](https://github.com/vitessio/vitess/pull/7277)

from `Build Environment Changes` section is redundant. 
It is listed as `VTGate / MySQL compatibility`.

**Comment by eseokoh** [comment]
VReplication section has a redundancy:
- vtctl: Add missing err checks for VReplication v2 #7361
- [vtctl] Add missing err checks for VReplication v2 #7361

**Comment by askdba** [comment]
@eseokoh thanks for pointing this out. We have generated updated release notes and will work on this to correct any mistakes over the official Release Notes. Please note we're continuously working to improve both the release and release process to avoid such incidents. 

```## Bug fixes
### Build/CI
 * make docker_local: fix missing mysql_server package #7213
### Cluster management
 * Backport: [vtctld] Fix accidentally-broken legacy vtctl output format #7292
### Query Serving
 * Fixes vtgate which throws an error in case of empty statements #6947
 * fix #7048 for SelectNone in StreamExecute route engine #7050
 * Make sure to check all GROUP BY columns #7080
## Documentation
### Other
 * Backport and fix tags for examples/operator #7358
## Enhancement
### Build/CI
 * [vtctldclient] vtctldclient generator #7238
### Other
 * Add s3 server-side encryption and decryption with customer provided key #7088
### Query Serving
 * Unnest simple subqueries #6831
 * Manage read your own writes system settings #6871
 * Adds Planning and Parsing Support for Create Index of MySQL 5.7 #7024
 * Support for hex & shard in vindex query #7044
 * Use distinct primitive to solve more queries #7047
 * Adds Planning and Parsing Support for Create View of MySQL 5.7 #7060
 * Adds Planning and Parsing Support for Create Database of MySQL 5.7 #7068
 * separate sub-query and derived table into different structs #7081
 * Adds Planning and Parsing Support for Alter Database of MySQL 5.7 #7086
 * Merge SelectDBA routes when possible #7140
 * Adds support for all the rails queries using information_schema #7143
 * Adds Planning and Parsing Support for Drop Table, Drop View and Alter View #7178
 * Redact password #7198
 * Adds Planning and Parsing Support for Alter Table #7199
## Feature Request
### Cluster management
 * Initial implementation of vtctld service #7128
 * Add FindAllShardsInKeyspace to vtctldserver #7201
### VTAdmin
 * Initial vtadmin-api, clusters, and service discovery #7187
 * [vtadmin] Add cluster protos to discovery and vtsql package constructors #7224
## Internal Cleanup
### Cluster management
 * action_repository: no need for http.Request #7124
### Other
 * Update Vitess v8.0 images #7174
### Query Serving
 * Convert usages of DDL struct to DDLStatement interface #7096
 * Adds Planning and Parsing Support for Drop Database of MySQL 5.7 #7098
 * Refactor Code to create a separate struct for CREATE TABLE #7116
 * Refactor Code to create a separate struct for ALTER VSCHEMA #7173
## Other
### Build/CI
 * Backport: Testing version upgrade/downgrade path from/to 8.0 #7323
### Cluster management
 * [vtctld] Fix accidentally-broken legacy vtctl output format #7285
### Other
 * Rewrite joins written with the USING construct #6660
 * enable client session tracking in mysql protocol #6783
 * Allow multiple blacklists for master #6816
 * Load Data From S3 #6823
 * show columns from table_name targeted like select queries #6825
 * VReplication: refactored and enhanced support for JSON columns #6829
 * online-DDL: migration uses low priority throttling #6830
 * Update `operator.yaml` backup engine description  #6832
 * Docker - upgrade to Debian Buster #6833
 * Updating azblob to remove directory after removing backup #6836
 * VReplication: Set time zone to UTC while streaming rows #6845
 * Materialization and character sets: Add test to verify/demo a workaround for charset issues while using string functions in filters #6847
 * Add timeout for mysqld_shutdown #6849
 * healthcheck: should receive healthcheck updates from all tablets in cells_to_watch #6852
 * Workflow listall with no workflows was missing newline #6853
 * Adding @shlomi-noach to CODEOWNERS #6855
 * allow incomplete SNAPSHOT keyspaces #6863
 * download zookeeper 3.4.14 from archive site #6865
 * Adding MySQL Check Constraints #6869
 * Bump junit from 4.12 to 4.13.1 in /java #6870
 * Additional options support for SELECT INTO and LOAD DATA #6872
 * fix ListBackups for gcp and az to work with root directory #6873
 * Fixing some flaky tests #6874
 * Pulling bootstrap resources from vitess-resources #6875
 * Flaky test: attempt to fix TestConnection in go/test/endtoend/messaging #6879
 * VDiff: make enums comparable #6880
 * Stabilize test #6882
 * tablet streaming health fix: never silently skip health state changes #6885
 * Add owners to /go/mysql #6886
 * VDiff: add ability to limit number of rows to compare #6890
 * VReplication: Tool to diagnose vreplication issues in production #6892
 * Fix parsing of online-ddl command line options #6900
 * Online DDL: followups in multiple trajectories #6901
 * 7.0.3 Release Notes #6902
 * Add Rohit Nayak to maintainers #6903
 * healthcheck: use isIncluded correctly to fix replica/rdonly routing bug #6904
 * [Java] Bump SNAPSHOT version to 9.0 after Vitess release 8.0 #6907
 * OnlineDDL bugfix: make sure schema is applied on tablet #6910
 * Fixes a bug in Load From statement #6911
 * query consolidator: fix to ignore leading margin comments #6917
 * vttablet: ignore errors if extracted gh-ost binary is identical to installed binary #6928
 * VReplication: Don't update tx timestamp on heartbeat #6930
 * Route INFORMATION_SCHEMA queries #6932
 * Change dependencies for lite builds #6933
 * Return to re-using server IDs for binlog connections #6941
 * VDiff/Tablet Picker: fix issue where vdiff sleeps occasionally for tablet picker retry interval #6944
 * [Forward Port] #6940 - Fix error handling in olap mode #6949
 * Fix unit test error #6953
 * Set Global  #6957
 * 8_0_0 Release Notes #6958
 * Truncate logged query in dbconn.go. #6959
 * Fixing the 5.6 builds of vitess/lite #6960
 * Vitess Slack Guidelines v1.0 #6961
 * Binary PK: fix bug where padding of binary columns was being done incorrectly #6963
 * Set udv allow more expressions #6964
 * Adding curl as dependency #6965
 * Pin mariadb to use mariadb-server-10.2 #6966
 * Pad non-fractional part of DECIMAL type #6967
 * Allow INSERT with all defaults #6969
 * Vtctld Workflow Command: fix error state in Workflow Show #6970
 * Fix comment typo #6974
 * Allow YEAR column type with length specified #6975
 * Bug which caused the connection to not close in case of error writing an error packet #6977
 * Healthcheck: add unit test for multi-cell replica configurations #6978
 * VStreamer Field Event: add allowed values for set/enum  #6981
 * Retry Execute when reserved connection is lost #6983
 * Correctly initialize the TabletType stats #6989
 * use ephemeral buffer when reading rows #6990
 * VReplication E2E Tests: Refactored  tests for readability and attempting to fix flakiness #6991
 * Make VReplication relay log size & rows configurable. #6992
 * Fix unit test error #6993
 * allow time_zone in reserved connection #6998
 * VRepl/Tablet Picker: improve observability of selected tablet #6999
 * Add DDL parser support for FULLTEXT indexes. #7001
 * Adds better error when there are timeouts in resource pools #7002
 * Replace vitess:base with vitess:lite images for docker-compose services #7004
 * Online DDL: cancel running migrations executed by another tablet #7006
 * Improved support for UNION #7007
 * Vtctld Workflow command: minor fixes #7008
 * Update docker-compose examples to take advantage of improvements in Vitess #7009
 * vttablet: fast and reliable state transitions #7011
 * VDiff: lock keyspace while snapshoting, restart target in case of errors #7012
 * fix flakey TestParallelRunnerApprovalFirstRunningSecondRunning test #7014
 * Extend comparability of EvalResult to support hash codes #7016
 * use proto equal method  #7017
 * MoveTables: validate that source tables exist, move all tables  #7018
 * Fix COM_STMT_EXECUTE packet decode #7020
 * Updates to Contacts section as Reporting #7023
 * Create pull_request_template #7027
 * UNION DISTINCT support on vtgate #7029
 * backup: Use pargzip instead of pgzip for compression. #7037
 * vttablet: don't shutdown on too many connections #7039
 * Adding `ddl_strategy` session variable #7042
 * Allow custom image tags in compose #7043
 * Online DDL: ddl_strategy session variable and vtctl command line argument #7045
 * Refactoring of plan building #7054
 * log sql which exceeds max memory rows #7055
 * clean up plan building test cases #7057
 * Fixed pull request template path #7062
 * Removing online ddl query hint from ALTER TABLE #7069
 * New VReplication workflows cli UX. Allow reads/writes to be switched independently #7071
 * Support statsd for vitess #7072
 * vschema: support back-quoted names #7073
 * vtgate -ddl-strategy flag renamed to -ddl_strategy #7074
 * Support CREATE, DROP statements in ApplySchema and online DDL #7083
 * Fix regression in /healthz #7090
 * VReplication: Handle comment statement type in vstreamer #7092
 * Update maintainers of Vitess #7093
 * Updating Email Address #7095
 * OnlineDDL: "cancel-all" command to cancel all pending migrations in keyspace #7099
 * Restore SHOW SCHEMAS support; fixes #7100 #7102
 * IsInternalOperationTableName: see if a table is used internally by vitess #7104
 * Update morgo changes #7105
 * Adds timeout to checking for tablets. #7106
 * remove deprecated vtctl commands, flags and vttablet rpcs #7115
 * Allows for vttestserver and vtcombo to respond to VtGateExecute. #7121
 * Fixes comment to mention the existence of reference tables. #7122
 * Add vtctl to make install-local #7125
 * Move PR template to .github directory #7126
 * go mod tidy #7137
 * VReplication e2e: Fine tuned test to reduce flakiness and added more logging to debug future flakiness  #7138
 * Support for lock and unlock tables #7139
 * Updating Java unit tests for JDK9+ compatability #7144
 * Support `vtctl OnlineDDL <keyspace> show <context>` #7145
 * Fix missed rename in onlineddl_test #7148
 * Allow system table to be set as default database #7150
 * Table lifecycle: skip time hint for unspecified states #7151
 * Normalizing Online-DDL queries #7153
 * VStreamer: Ignore temp tables created by onlineddl  #7159
 * Move auto_increment from reserved to non reserved keyword #7162
 * add support for unary expression in order by column #7163
 * Skip query rewriting for dual table #7164
 * Add only expr of aliasedExpr to weightstring function #7165
 * Move allow_alias option in MySqlFlag enum to precede the aliased IDs #7166
 * Online DDL endtoend tests to support MacOS #7168
 * SwitchWrites bug: reverse replication workflows can have wrong start positions #7169
 * Online DDL: ddl_strategy=direct #7172
 * healthcheck: healthy list should be recomputed when a tablet is removed #7176
 * Fix trivial typo #7179
 * Add Version to Bootstrap Image #7182
 * Fix broken package ref in UBI docker build #7183
 * Refactor Show plans #7185
 * add sequence table support for vtexplain #7186
 * Convert CentOS extra packages installation to yum instead of downloading #7188
 * vttablet: debug/env page to change variables in real-time #7189
 * Updated pull request template to add more clarity #7193
 * Show privilege support #7194
 * cache only dml and select plans #7196
 * Add @ajm188 + @doeg to CODEOWNERS for vtctld service files #7202
 * Update Golang to 1.15 #7204
 * Online DDL executor database pool size increase #7206
 * doc: fix comment for IDPool #7212
 * improve-log: FAILED_PRECONDITION #7215
 * Online DDL: DROP TABLE translated to RENAME TABLE statement #7221
 * Add @ajm188 + @doeg as vtadmin codeowners #7223
 * added readme for the demo example #7226
 * Add unit test case to improve test coverage for go/sqltypes/result.go #7227
 * fix meta data related operation hang when zk down #7228
 * DropSources: change table rename logic #7230
 * MoveTables: delete routing rules and update vschema on Complete and Abort #7234
 * Use `context` from Go's standard library #7235
 * Fix all occurrences of `fmt.Sprint(x)` where x is `int` #7244
 * Add linter configuration #7247
 * V2 Workflow Start: wait for streams to start and report errors if any while starting a workflow #7248
 * local docker: copy examples/common into /vt/common to match MoveTables user guide #7252
 * Fix incorrect comments #7257
 * Adding @@session_uuid to vtgate; used as 'context' by Online DDL #7263
 * goimport proto files correctly #7264
 * Cherry pick version of #7233 for release-9.0 #7265
 * [9.0] don't try to compare varchars in vtgate #7271
 * Default to false for system settings to be changed per session at the database connection level #7299
 * Backport 7297: VStreamer Events: remove preceding zeroes from decimals in Row Events #7340
 * Update Java version to 9.0 #7369
### Query Serving
 * Allow table_schema comparisons #6887
 * Improve database ddl plan #7034
 * vtgate: route using vindex for composite IN clause #7049
 * OnlineDDL: request_context/migration_context #7082
 * Online DDL:  ddl_type column #7097
### VReplication
 * Do vschema_customer_sharded.json before create_customer_sharded.sql. #7210
 * Backport #7276: Vreplication V2 Workflows: rename Abort to Cancel #7339
 * Backport #7255: VReplication DryRun: Report current dry run results for v2 commands #7345
 * Backport 7275: VReplication: Miscellaneous improvements #7349
 * Backport 7342: Workflow Show: use timeUpdated to calculate vreplication lag #7354
 * Backport 7361: vtctl: Add missing err checks for VReplication v2 #7363
### VTAdmin
 * [vtadmin-web] The tiniest possible first implementation of vtadmin-web #7218
 * [vtadmin-api] Add static file service discovery implementation #7229
 * [vtadmin-web] Query vtadmin-api from vtadmin-web with fetch + react-query #7239
## Performance
### Query Serving
 * Optimise struct field layout #7052
 ```

**Comment by askdba** [comment]
Closing this issue as it's fixed in the above PR #8295.
